lib/tools/system-calls.js

```diff
@@ -460,7 +460,9 @@ systemCallMethods.reboot = async function () {
     if (booted === '1') {
       return;
     } else {
-      log.errorAndThrow('Waiting for reboot this takes time');
+      let msg = 'Waiting for reboot. This takes time';
+      log.debug(msg);
+      throw new Error(msg);
```

Comment:
This is explicitly _not_ using that (see the diff). Otherwise you get a stack trace for every throw, and the log looks like it's full of errors instead of expected behaviour..